packages/drivers/odsp-driver/src/odspCache.ts

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-
+import { strict as assert } from "assert";
```

Comment:
Please use `assert` from common-utils to avoid bundle size increases.  Would be nice if we had a lint rule to block this or something...

Reply:
Good to know! Thanks